README.md

```
@@ -1,86 +1,245 @@
+- **Auto-deactivation** — when the bot is removed from a group, the group is automatically marked inactive and stops receiving messages.
+- **In-memory group cache** — group data is cached in memory for fast access without repeated DB queries.
 
-## Installation
+---
 
-Ensure you have [pnpm](https://pnpm.io/) installed. You can install it globally using npm:
+## Architecture Overview
 
-```bash
-npm install -g pnpm
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)